lib/linguist/languages.yml

```diff
@@ -2879,6 +2879,7 @@ Python:
   ace_mode: python
   color: "#3572A5"
   extensions:
+  - .BUILD
```

Comment:
Don't put your extension to first place. First extension in a list is a primary extension of files for language.